test/quantization/test_quantize_fx.py

```diff
@@ -332,7 +333,8 @@ def forward(self, x):
 
         m = M().eval()
         m = symbolic_trace(m)
-        qconfig_dict = {'': default_qconfig, 'conv2': None}
+        qconfig_dict = {"": default_qconfig,
+                        "module_name": [("conv2", None)]}
```

Comment:
Is there a reason to prefer ("conv2", None) to  "conv2": None?  Wouldnt a dict be more natural here?

Reply:
yeah, i agree. the reason we use ("conv2", None) is because "module_name_regex" needs to be matched in order and we want to keep the type for all configurations consistent.